dspace-oai/src/main/java/org/dspace/xoai/util/DateUtils.java

```diff
@@ -96,7 +111,8 @@ public static Date parse(String date)
 
     public static Date parseFromSolrDate(String date)
     {
-        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.getDefault());
+        SimpleDateFormat format = new SimpleDateFormat(
+                "yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.getDefault());
```

Comment:
This is one of the points we should use Zulu Time, but do use the configured default Timezone.